payroll/views/hr_payslip_line_views.xml

```diff
@@ -105,11 +119,8 @@
     >
         <field name="name">Payslip Computation Details</field>
         <field name="res_model">hr.payslip.line</field>
-        <field name="view_mode">tree,form</field>
-        <field name="domain">[]</field>
-        <field name="context">
-            {'default_slip_id': active_id,'search_default_slip_id': active_id}
-        </field>
+        <field name="view_mode">tree,pivot,form</field>
+        <field name="domain">[('slip_id', 'in', active_ids)]</field>
```

Comment:
Not sure what this change do?

Reply:
the domain one

Reply:
Select multiple payslips and click Action - Payslip Computation Details to see all the lines of all the selected payslips.